docs/docs/getting-started/cli.md

```diff
@@ -0,0 +1,247 @@
+# Getting started with Tracetest using the CLI
+
+The CLI offers an **install wizard** that helps with the process. It helps configure Tracetest itself and all the tools needed to observe your application.
+
+## Installing the Tracetest CLI
+
+Use the install wizard to install Tracetest locally using Docker Compose or to a local or remote Kubernetes cluster.
+
+It installs all the tools required to set up the desired environment and creates all the configurations, tailored to your case.
+
+Every time we release a new version of Tracetest, we generate binaries for Linux, MacOS, and Windows. Supporting both amd64, and ARM64 architectures, in `tar.gz`, `deb`, `rpm` and `exe` formats.
+
+You can find the latest version [here](https://github.com/kubeshop/tracetest/releases/latest).
+
+### Linux/MacOS
+
+Tracetest CLI can be installed automatically using the following script:
+
+```sh
+curl -L https://raw.githubusercontent.com/kubeshop/tracetest/main/install-cli.sh | bash
+```
+
+It works for systems with Hombrew, `apt-get`, `dpkg`, `yum`, `rpm` installed, and if no package manager is available, it will try to download the build and install it manually.
```

Comment:
```suggestion
It works for systems with Homebrew, `apt-get`, `dpkg`, `yum`, `rpm` installed, and if no package manager is available, it will try to download the build and install it manually.
```